win_build.ps1

```diff
@@ -56,6 +56,8 @@ echo $msBuild
 $env:GOOS="windows"
 $env:GOARCH=$arch
 
+go mod download
```

Comment:
this shouldn't be required. all go commands with modules support should automatically get the dependencies. Go mod download just gets them and caches locally, but i'm not sure we should do that.